multicluster/test/e2e/service_test.go

```diff
@@ -252,3 +277,13 @@ func teardownGateway(t *testing.T, data *MCTestData) {
 		}
 	}
 }
+
+func (data *MCTestData) probeFromCluster(clusterName string, url string) error {
```

Comment:
Why do you add this back to probe Service from Node? it's not correct. 
Please refer to `probeServiceFromPodInCluster`

Reply:
Thanks. Changed.